src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterChecksumChecker.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.sendletter.services;
+
+import org.apache.commons.lang.NotImplementedException;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sendletter.model.Letter;
+
+@Component
+public class LetterChecksumChecker {
```

Comment:
How about `LetterChecksumGenerator` or something like this? Not sure about the `Checker` part.

Reply:
This class is still not implemented, but it would be good to add a test class for it as part of this pull request, with some test marked as ignored.